src/app/src/main/kotlin/org/imperial/mrc/hint/controllers/LoginController.kt

```diff
@@ -63,4 +65,10 @@ class LoginController(
     {
         return oauth2RegisterRedirect()
     }
+
+    @GetMapping("/sso")
+    fun isSSOLoginMethod(): ResponseEntity<String>
```

Comment:
What's the reason for supporting non-SSO? I thought our intention was to switch over completely to using ADR to authenticate HINT users via their resource server? What configurations won't do that?
If it's to allow testing without exposing the secret, we should be able to do a fetch of the secret from the vault in the CI script. Or is it to allow us an easy way back if it doesn't work out?!